src/chapters/control-flow/conditionals.rst

```diff
@@ -0,0 +1,292 @@
+``case`` for ``4`` and assigns the value "Thursday" to the variable
+``day``, it proceeds to execute every statement in every case that
+follows, all the way through the ``default`` case. So the ``String``
+that ends up being printed will reflect the last executed statement in
+the ``switch`` block.
+
+Along similar lines, consider this variation on the code block above:
+
+.. sourcecode:: java
+   :linenos:
+
+   import java.util.Scanner;
+
+   public class DayPrinter {
+      public static void main(String[] args) {
+
+         System.out.println("Enter an integer: ");
+         Scanner in = new Scanner(System.in);
+         int dayNum = in.nextInt();
+
+         String day;
+         switch (dayNum) {
+            case 0:
+               day = "Sunday";
+            case 1:
+               day = "Monday";
+            case 2:
+               day = "Tuesday";
+            case 3:
+               day = "Wednesday";
+            case 4:
+               day = "Thursday";
+            case 5:
+               day = "Friday";
+            case 6:
+               day = "Saturday";
+               break;
+            default:
+               day = "Int does not correspond to a day of the week";
+         }
+         System.out.println(day);
+      }
+   }
+
+
+Here, we have a ``break``statement in ``case 6`` after ``day = "Saturday";``. 
+If the user enters ``4``, the execution will fallthrough until it reaches that
+``break`` statement and "Saturday" is printed instead of "Thursday".
+
+References
+----------
+
+-  `The if-then and if-then-else Statements
+   (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/nutsandbolts/if.html>`__
+-  `The switch Statement
+   (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html>`__
+
+Check Your Understanding
+-------------------------
+
+.. admonition:: Question
+
+   When does fallthrough occur in Java?
+
+   #. Omitting an ``else`` clause from a conditional.
+   #. Omitting an ``else`` clause from switch statement.
+   #. Omitting a ``default`` case from a ``switch`` statement.
+   #. Omitting a ``break`` line from a ``switch`` statement.
+
+.. admonition:: Question
+
+   .. sourcecode:: java
+      :linenos:
+
+      import java.util.Scanner;
+
+      public class QuizQuestion {
+         public static void main(String[] args) {
+
+            System.out.println("Are you a space cadet? yes or no");
+            Scanner in = new Scanner(System.in);
+            String response = in.next();
+
+            switch (response) {
+               case "yes":
+                  System.out.println("Greetings cadet.");
+               case "no":
+                  System.out.println("Greetings normie.");
+               default:
+                  System.out.println("Are you an alien?");
+            }
+         }
+      }
+
+   Given the code above, what prints if the user enters "no" after the prompt?
+
+   #. "Greetings cadet."
```

Comment:
Use code highlighting instead of quotes for the output.  Also, what in the world is a "normie"?  ;-)

Reply:
idk just coming up with content